experimental/perf/scripts/push_perf_results_helper.py

```diff
@@ -1,95 +1,38 @@
 import argparse
+import json
 import os
 import re
-
 import wandb
 
 
-def parse_log(log_path):
-    """Parses a performance test log file and extracts relevant metrics.
-
-    Args:
-        log_path (str): Path to the log file.
-
-    Returns:
-        dict: A dictionary of parsed metrics.
-    """
-    with open(log_path) as f:
-        log_content = f.read()
-
-    # Extract general information
-    steps_per_run = int(
-        re.search(r"# of steps in each run: (\d+)", log_content).group(1)
-    )
-    metrics_per_step = int(
-        re.search(r"# of metrics in each step: (\d+)", log_content).group(1)
-    )
-
-    # Extract timing information for the run
-    init_time = float(re.search(r"init_wandb\(\) time: ([\d.]+)", log_content).group(1))
-    log_metrics_time = float(
-        re.search(r"log_metrics\(\) time: ([\d.]+)", log_content).group(1)
-    )
-    finish_time = float(
-        re.search(r"finish_wandb\(\) time: ([\d.]+)", log_content).group(1)
-    )
-
-    # Extract system metrics
-    system_metrics = {}
-    for match in re.finditer(r"\{([^}]+)\}", log_content):
-        metrics = match.group(1).split(", ")
-        for metric in metrics:
-            key, value = metric.split(":")
-            system_metrics[key] = float(value)
-
-    # Consolidate all extracted data
-    metrics = {
-        "steps_per_run": steps_per_run,
-        "metrics_per_step": metrics_per_step,
-        "timing": {
-            "init_time": init_time,
-            "log_metrics_time": log_metrics_time,
-            "finish_time": finish_time,
-        },
-        "system_metrics": system_metrics,
-    }
-
-    print(metrics)
-    return metrics
-
+# Load JSON data from files
+def load_json(filename: str) -> dict:
+    with open(filename, "r") as f:
+        return json.load(f)
```

Comment:
probably don't even need this function given that it only loads a json string from file?